### PR TITLE
feat: Navigation Added

### PR DESCRIPTION
**Pull request by mdarif6**
Navigation Added to Component Library..

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Good naming conventions
Clean code
Good for approval ✅ 

Small suggestion: You can add images or gif in PR preview

File: components/navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div class="ha-content">
+      <header class="ha-header">
+        <div class="ha-logo">
+          <a href="#"
+            ><img src="/document/images/logo_navigation.png" alt=""
```
Comment (Shah-Arsalan) [inline components/navigation/navigation.html]: give alt value

File: components/navigation/navigation.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        </div>
+      </header>
+    </div>
+
+    <div class="ha-content-forsmall">
+      <header class="ha-header-forsmall">
+        <div class="logo-and-icon-for-small">
+          <div class="ha-logo">
+            <a href="#"
+              ><img src="/document/images/logo_navigation.png" alt=""
```
Comment (Shah-Arsalan) [inline components/navigation/navigation.html]: here too

File: components/navigation/navigation.css [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+}
+
+/* Header */
+
+.ha-header {
+  grid-area: header;
+  align-items: center;
+  justify-content: center;
+  display: flex;
+  background-color: rgb(235, 231, 231);
```
Comment (Shah-Arsalan) [inline components/navigation/navigation.css]: use color variable instead of hard coded value

**Review by Shah-Arsalan — APPROVED** [review]
Everything looks good . You can merge

File: components/navigation/navigation.html (hunk 2 above)
Comment (mdarif6) [reply components/navigation/navigation.html]: done

File: components/navigation/navigation.css (hunk 3 above)
Comment (mdarif6) [reply components/navigation/navigation.css]: done

**Pull request merged**